sys/include/net/gnrc/udp.h

```diff
@@ -54,6 +54,13 @@ extern "C" {
 #endif
 
 /**
+ * @brief   Default UDP port range for dynamic source port allocation
+ */
+#define GNRC_UDP_PORTRANGE_MIN (49152U)
```

Comment:
Ephemeral is the term used in the RFC and "Unix Network Programming, Volume 1: Networking APIs: Socket and XTI", Prentice Hall, 1998". Hence, for me it is a well defined term. But "SRC_PORTRANGE" would work for me, if you insist.